apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss

```diff
@@ -72,12 +71,12 @@
 
                 p{
                     font-size: 14px;
-                    font-family: 'opensans-semibold'; 
+                    font-family: 'opensans-semibold';
```

Comment:
Please use font variable

Reply:
Font variable is not working. It takes time for R&D about font variable